vignettes/covariance.Rmd

```diff
@@ -40,37 +43,30 @@ introduce these here.
 
 ## Covariance and Correlation Matrices
 
-The symmetric and positive definite covariance matrix
-\[
```

Comment:
I think this was a bit better before, using `\[` instead of `$$` as well as having that on a separate line instead of at the end of the previous line. If not too much work, can we revert to that for the large (not inline) equations?

Reply:
updated in commit https://github.com/openpharma/mmrm/pull/80/commits/1c644b82b2785bb85156e188572734f3c8586719

Reply:
thanks Ya!